src/remote/configuration/toml.py

```diff
@@ -295,25 +297,32 @@ def load_config(self, workspace_root: Path) -> WorkspaceConfig:
                     directory=connection.directory or self._generate_remote_directory_from_path(workspace_root),
                 )
             )
-        ignores = SyncIgnores(
+        ignores = SyncRules(
             pull=_get_exclude(merged_config.pull, workspace_root),
             push=_get_exclude(merged_config.push, workspace_root),
             both=_get_exclude(merged_config.both, workspace_root) + [WORKSPACE_CONFIG],
         )
 
+        includes = SyncRules(
+            pull=merged_config.pull.include if merged_config.pull and merged_config.pull.include else [],
+            push=merged_config.push.include if merged_config.push and merged_config.push.include else [],
+            both=merged_config.both.include if merged_config.both and merged_config.both.include else [],
```

Comment:
You can change it to `push=merged_config.push.include if merged_config.push else [],` since if merged_config.push exists merged_config.push.include is guaranteed to exist and be a list

Reply:
Done.